crates/sui-framework/packages/sui-framework/sources/crypto/nitro_attestation.move

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::nitro_attestation;
+
+use sui::clock::{Self, Clock};
+
+public struct NitroAttestationDocument has store, copy, drop  {
+    module_id: vector<u8>,
+    timestamp: u64,
+    digest: vector<u8>,
+    pcrs: vector<vector<u8>>,
```

Comment:
there is some meaning to the order, no? if simple, let's document it, else let's define this type explicitly

Reply:
added

Reply:
are we sure that PCR6 for example won't be added in the future? if not, should we store a vector of PCR { index: u8, data: vector<u8>}

Reply:
revised to have the pcrs getter function to return a vector of PCREntry {index, vector itself}